macros/sql/nullcheck_table.sql

```diff
@@ -1,8 +1,8 @@
 {% macro nullcheck_table(schema, table) %}
 
-  {% set cols = adapter.get_columns_in_table(schema, table) %}
+  {% set cols = adapter.get_columns_in_relation(this)) %}
```

Comment:
@Nerdenberger I think you have an extra close paren here:
```suggestion
  {% set cols = adapter.get_columns_in_relation(this) %}
```

Reply:
in addition the extra paren, we'll want to construct a [Relation object](https://docs.getdbt.com/reference#section-creating-relations) here from the `schema` and `table` arguments passed into the macro.

This is kind of tricky because Relations should be constructed with a database name, but the `nullcheck_table` macro doesn't accept a database. @jtcohen6 do you think we should add an arg? Unsure how much this macro is used these days

Reply:
I think we could update line 6 below to be `from {{relation}}` and thereby accept a (default) database above. The [`nullcheck` macro](https://github.com/fishtown-analytics/dbt-utils/blob/master/macros/sql/nullcheck.sql) just accepts a Jinja list of columns, it doesn't do any introspective work itself.

To answer your larger question, tbqh I've never seen this macro used, and it looks like it was last touched a while ago.